packages/components/Menu/src/MenuCallout/MenuCallout.types.ts

```diff
@@ -0,0 +1,22 @@
+import type { ICalloutProps, ICalloutTokens } from '@fluentui-react-native/callout';
+
+export const menuCalloutName = 'MenuCallout';
+
+// Support for anchorRect and beakWidth will come at a later time.
```

Comment:
Does this mean like a separate follow up PR for Menu?

Reply:
No this was forked out from MenuPopver based on the conversation here: https://github.com/microsoft/fluentui-react-native/pull/2715#discussion_r1157440435 

So this comes from the Menupopver tokens that comes from Callout which is for win32 and win32 owners might have planned something on this so i transported the comments here.